GSA/digitalgov.gov
# Creating the August 2024 USWDS event page

## Summary

Creating the event page for the August 2024 USWDS Monthly Call

## Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/brunerae-uswds-aug-mc/)



## How To Test

1. Navigate to "Events" 
2. Click on the August 2024 Monthly Call
3. Verify information matches the [Event Brief.](https://docs.google.com/document/d/186KgWqvof7utMdzGaPySO3cRMrP5C8gT18kdV5iLMq0/edit#heading=h.3xsjlmyr1tug)
